boards/board-stub.dts

```diff
@@ -0,0 +1,12 @@
+/*
```

Comment:
Move to dts/common and include skeleton.dtsi

Reply:
@tejlmand agree?

Reply:
The main requirement from my side is to get skeleton.dtsi in there. The location is negotiable

Reply:
> The main requirement from my side is to get skeleton.dtsi in there. The location is negotiable

Can you please elaborate why skeleton.dtsi is a requirement for this particular case? I just want to understand the reasons before making the change.

Reply:
1. I want to keep consistency
2. An empty root node is disallowed by the DT spec -- we can be flexible with missing things like `model` and `compatible`, but I at least want to have `#address-cells` and `#size-cells` defined with sensible defaults. See Table 3.2 Root Node Properties in the DT spec

Reply:
> @tejlmand agree?

From dts viewpoint, then `dts/common` looks better, but from users point of view where the `board-stub.dts` functions as a board's dts file when a board is missing a dts file, then I believe the boards folder seems most appropriate.

Just like regular boards has their dts file as `boards/<arch>/<board>/<boardname>.dts`, then the stub is in the board hierarchy.

I do not have a strong opinion and can accept both locations, but based on above reasoning then I have a preference for the `boards` location.

Reply:
OK @tejlmand -- how about boards/common/board-stub.dts? Putting this right in boards/ feels like showing everyone who wants to look at zephyr's boards something that 99.999% of them will not care about.

Reply:
Perhaps just `boards/common/stub.dts`?

Reply:
followed @henrikbrixandersen suggestion, and skeleton.dtsi is included.